feat(no-empty-fields): Add `ignoreProperties` option (#1186)
JoshuaKGoldberg/eslint-plugin-package-json: docs/rules/no-empty-fields.md

	}
}
```

## Options

### `ignoreProperties`

Pass an array of top-level package properties to ignore.

> When provided, any errors related to the properties won't cause the rule to report a violation.

Reviewer: [Docs] Nit: "errors" are a by-product of rule report violations, not the other way around. ✨ 

```suggestion
When provided, any of those properties being empty won't cause the rule to report a violation.
```

> When provided, any errors related to the properties won't cause the rule to report a violation.

Reviewer 2: I think @JoshuaKGoldberg commented on this, but I'm not seeing it anymore.  I generally agree with what he said, though.  Errors aren't errors if they're not reported 😁
Perhaps something along the lines of 

```suggestion
When provided, the rule won't report violations for the specified properties.
```
